src/idl_gen_python.cpp

```diff
@@ -412,6 +417,51 @@ class PythonGenerator : public BaseGenerator {
     code += Indent + Indent + "return None\n\n";
   }
 
+    // Get the value of a vector's struct member.
+  void GetMemberOfVectorOfUnion(const StructDef &struct_def,
+                                 const FieldDef &field, std::string *code_ptr,
+                                 ImportMap &imports) const {
+    auto &code = *code_ptr;
+    auto vectortype = field.value.type.VectorType();
+    GenReceiver(struct_def, code_ptr);
+    std::string return_type = "flatbuffers.table.Table";
+
+    // bool is_native_table = vectortype.base_type == "*flatbuffers.Table";
+    code += namer_.Method(field);
+    bool is_native_table = TypeName(field) == "*flatbuffers.Table";
+    ImportMapEntry import_entry;
+    if (is_native_table) {
+      import_entry = ImportMapEntry{ "flatbuffers.table", "Table" };
+    } else {
+      return_type = TypeName(field);
+      if (!parser_.opts.one_file) {
+        import_entry = ImportMapEntry{ GenPackageReference(field.value.type),
+                                     TypeName(field) };
+      }
+    }
+
+
+    if (parser_.opts.python_typing) {
+      code += "(self, j: int) -> Optional[" + return_type + "]";
+      imports.insert(ImportMapEntry{ "typing", "Optional" });
+      if (!parser_.opts.one_file) {
+        imports.insert(import_entry);
+      }
+    } else {
+      code += "(self, j)";
+    }
+    code += ":" + OffsetPrefix(field);
+    code += Indent + Indent + Indent + "x = self._tab.Vector(o)\n";
```

Comment:
fwiw there is an alternative pattern used here of `GenIndents()` which I always liked a bit better here but its totally optional this file uses both pretty interchangably: https://github.com/google/flatbuffers/blob/master/src/idl_gen_python.cpp#L846

Reply:
Ah, you are using that later, either way is cool